app/javascript/tests/patient/laboratory/FirstPositiveLaboratory.test.js

```diff
@@ -1,37 +1,40 @@
 import React from 'react';
 import { shallow } from 'enzyme';
 import { Button } from 'react-bootstrap';
+import ReactTooltip from 'react-tooltip';
 import moment from 'moment';
 
 import FirstPositiveLaboratory from '../../../components/patient/laboratory/FirstPositiveLaboratory';
 import { mockLaboratory1 } from '../../mocks/mockLaboratories';
 
 import LaboratoryModal from '../../../components/patient/laboratory/LaboratoryModal';
 
-function getWrapper(lab) {
-  return shallow(<FirstPositiveLaboratory lab={lab} />);
+function getWrapper(lab, disableDelete) {
```

Comment:
picky but can you update this to use additionalProps and only pass in true for the last test? something like
```
function getWrapper(additionalProps) {
  return shallow(<FirstPositiveLaboratory lab={mockLaboratory1} {...additionalProps} />);
}
```

then for the first two tests:
```
const wrapper = getWrapper({ lab: null });
```

and then the last two tests:
```
const wrapper = getWrapper({ disableDelete: true });
```

Reply:
[9d0764e](https://github.com/SaraAlert/SaraAlert/pull/1205/commits/9d0764e18fcec02f59b0d8ce888ddf4e029a0e36)